### ISSUE TITLE
App crashes when using experimental mpv player

### ISSUE THREAD

**Issue by Grummm** [jarnedemeulemeester/findroid#805]
**Describe the bug**
Since version 0.15.0 when playing any media with the experimental mpv player the app crashes. 

**To Reproduce**
1. In video player setting enable mpv player.
2. Try to play any media on your jellyfin server.

**Expected behavior**
It should just play the media with mpv.

**Device info (please complete the following information):**
 - Device: samsung galaxy a52s
 - Android version: 14
 - One UI version: 6.1
 - App version: 0.15.0
 - Jellyfin version: 10.9.8

**Comment by jmqm** [comment]
Unsure if the same issue, but mine crashes consistently when playing downloaded media. Previously it crashed often, now it crashes all the time. Before when I turn on offline mode it rarely crashed, now it doesn't matter.

MPV settings:
mediacodec
GPU
audiotrack

Findroid 0.15.0 (26)

**Comment by Natanel-Shitrit** [comment]
Happens for me too, only downloaded media.

**Comment by Grummm** [comment]
Doesn't matter for me, happens with both downloaded and streamed media. Haven't had the chance to test it on other devices though.

And what do you mean with the original issue? Haven't been using findroid for that long but never had any issues playing downloaded media with mpv.

**Comment by Jcuhfehl** [comment]
> And what do you mean with the original issue? Haven't been using findroid for that long but never had any issues playing downloaded media with mpv.

I don't think this was ever an issue on a release, but during initial development of download playback there was a similar crash which was (in some weird way) fixed by adding a 1ms delay.
That delay was removed in this release because in both my and Jarne's testing it wasn't needed anymore, but it might still be needed after all.

**Comment by formeo14** [comment]
Thanks. I thought that it was a general issue with the new version. 
I did not think of the mpv player.

**Comment by jarnedemeulemeester** [comment]
Should be fixed with c3539fa9bb24e1faecb2a96e97a5989438eaa3fa
I will do some more testing before releasing a new version since there is still one other crash related to mpv

**Comment by kpdh** [comment]
I have the same.issue.specifically with downloaded media. Any update on when we can expect a push to the play store?

**Comment by Grummm** [comment]
Damn you're fast!
